src/ape_compile/_cli.py

```diff
@@ -35,27 +35,34 @@ def cli(cli_ctx, file_paths, use_cache, display_size):
     a project is loaded. You do not have to manually trigger a recompile.
     """
 
-    if not file_paths and cli_ctx.project.sources_missing:
-        contracts_dir = cli_ctx.project.config.contracts_folder
+    if not file_paths and cli_ctx.project_manager.sources_missing:
+        contracts_dir = cli_ctx.project_manager.config_manager.contracts_folder
```

Comment:
`cli_ctx` should have `config_manager` in it (can it inherit `ManagerAccessBase`?)